### ISSUE TITLE
xUnit1047 analyzer diagnostic incorrectly reported

### ISSUE THREAD

**Issue by jasonwoods-7** [xunit/xunit#2914]
Description:

The following code produces an `xUnit1047` analyzer diagnostic:

```csharp
public class Foo {
	public Foo(params object[] args) { }
}

public class TestClass {
	public void TestMethod() {
		var foo = new Foo(new object());
	}
}
```

Expected behavior:

Since this isn't part of a `TheoryDataRow`, it seems this diagnostic shouldn't be reported.

Repo:

I've added a failing unit test here jasonwoods-7/xunit.analyzers@ada7c66

**Comment by bradwilson** [comment]
Wow, that's a dumb bug on my part. 😭

**Comment by bradwilson** [comment]
Available in `1.13.0-pre.2` https://xunit.net/docs/using-ci-builds
